metricbeat/module/elasticsearch/node_stats/data.go

```diff
@@ -0,0 +1,60 @@
+package node_stats
```

Comment:
yes, because we have also other packages with this kind of names. @7AC can we configure it to ignore these errors?